plugin_player.py

```diff
@@ -0,0 +1,96 @@
+#file and library helpers
+import importlib
+import os
+import sys
+
+#helper classes for a PluginPlayer interface
+import pluginplay as pp
+from plugin_manager import PluginManager
+from tree_manager import TreeManager
+from node_widget_manager import NodeWidgetManager
+from node_manager import NodeManager
+from utility_manager import UtilityManager
+
+#kivy helpers
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.popup import Popup
+from kivy.uix.image import Image
+
+#image resizer
+from PIL import Image as PILImage
+
+
+#Class defining the running app
+class PluginPlayer(App):
+
+    #build the main window from the kv file
+    def build(self):
+        self.popup = Popup()
+
+        #The app's module manager
+        self.mm = pp.ModuleManager()
+
+        #list of imported plugins and their modules
+        self.savedPlugins = []
+
+        #list of imported class types
+        self.imported_classes = []
+
+        #saved tree containing the nodes and modules to be ran
+        self.nodes = []
+        
+        #helper class handling addition/removal of nodes, deleting/running the tree
+        self.tree_manager = TreeManager(self)
+
+        #helper class handling the widget building for the node configuration
+        self.node_widget_manager = NodeWidgetManager(self)
+
+        #helper class handling the linking of inputs, submods, property types between modules
+        self.node_manager = NodeManager(self)
+
+        #helper class handling the loading, deleting, and viewing of plugins and their modules
+        self.plugin_manager = PluginManager(self)
+
+        #helper class handling browsing, imported class types, and importing new classes
+        self.utility_manager = UtilityManager(self)
+
+        #build the main application from the kivy script file
+        build = Builder.load_file('plugin_player_setup.kv')
+
+        #add logo
+        tree_section = build.ids.right_section.ids.tree_section
+        self.create_image('NWCHEMEX.png', 'NWCHEMEX_icon.png', (200,200))
+        logo = Image(source='NWCHEMEX_icon.png', size_hint=(None, None), size=(200, 200), pos_hint={'x': .75, 'y': 0})
```

Comment:
```suggestion
        logo = Image(source='NWCHEMEX.png', fit_mode="fill", size_hint=(None, None), size=(200, 200), pos_hint={'right': 1, 'y': 0})
```
The [`fit_mode` option](https://kivy.org/doc/stable/api-kivy.uix.image.html#kivy.uix.image.Image.fit_mode) is probably the better way of handling the resizing. You might opt for `"contain"` instead of `"fill"`.